beancount_web/templates/charts/_chart_yearly_totals.html

```diff
@@ -0,0 +1,29 @@
+<script>
+    window.chartData.push({
+        label: "{{ label }}",
+        type:  "bar",
+        dateFormat: "YYYY", // #TODO: Why doesn't this work?
```

Comment:
@davidastephens Two reasons why this did not work properly:

1. `dateFormat` has to be nested into a dictionary called `options`, see https://github.com/aumayr/beancount-web/commit/be39f1ccd403ce5c898898975405a1fbe84f352b#diff-113bfa7a88d6e5731c07319321981c0fR5
2. The `YYYY`-format-string has to be implemented manually. I did choose to implement these format-strings myself, because libraries like `moment.js` are quite heavy and I need only very few of those. See https://github.com/aumayr/beancount-web/commit/be39f1ccd403ce5c898898975405a1fbe84f352b#diff-d7f3007e2206954d98f57e59551d9403R71